.github/workflows/bump.yaml

```diff
@@ -57,9 +57,14 @@ jobs:
         run: |
           echo "RELEASE_BRANCH=${{ github.ref_name }}" >> $GITHUB_ENV
 
-      - name: Bump chart version
+      - name: Set chart path
         working-directory: ./src/github.com/openshift-knative/knative-istio-authz-chart
-        run: go run hack/cmd/bumpistiochart/bumpistiochart.go --branch "${RELEASE_BRANCH}"
+        run: |
+          echo "ISTIO_AUTHZ_CHART_PATH=$(pwd)" >> $GITHUB_ENV
+
+      - name: Bump chart version
+        working-directory: ./src/github.com/${{ github.repository }}
```

Comment:
Does this point to S-O?

Reply:
yes

Reply:
is the repository of the workflow

Reply:
see line 40

Reply:
it's not checked out in the Go path, see line 40

Reply:
that needs to match where SO is checked out and it's not necessarily the same as the Go path

Reply:
Another approach would be to use ${{ github.workspace }} for the `--chart-metadata-path`

Reply:
GOPATH: ${{ github.workspace }}
GOPATH: /home/runner/work/serverless-operator/serverless-operator
So we checkout in the runners home dir it seems and the relative path is based on that.

Reply:
But it doesn't have to be like that, it is just a case that they match, sorry but I don't get why adding more variables to the line helps in any way

Reply:
Now, it is consistent with the other usage at line 92-94

Reply:
I said ok abov, we are in agreement :) I am just trying to understand the implementation :)